src/UseCases/ShowApplicationConfirmation/ShowApplicationConfirmationPresenter.php

```diff
@@ -6,13 +6,9 @@
 
 use WMDE\Fundraising\MembershipContext\Domain\Model\MembershipApplication;
 
-/**
- * @license GPL-2.0-or-later
- * @author Jeroen De Dauw < jeroendedauw@gmail.com >
- */
 interface ShowApplicationConfirmationPresenter {
 
-	public function presentConfirmation( MembershipApplication $application, string $updateToken ): void;
+	public function presentConfirmation( MembershipApplication $application, array $paymentData, string $updateToken ): void;
```

Comment:
For future PHPStan level increases, could you add the `array<string,mixed>` type to a docblock?